### PR TITLE
fix(tx-history): top menu transparency android

### PR DESCRIPTION
**Pull request by banklesss**
## Description / Change(s) / Related issue(s)

Top bar menu is not transparent in tx history screen on Android

##### Ticket

YOMO-1679

> [!NOTE]
> Please create the ticket if missing it.

### CODE REVIEW

**Review by michalszorad — APPROVED** [review]

File: apps/wallet-mobile/src/features/ListTxHistory/useCases/ListTxHistory/TxHistory.tsx [hunk 1]
```diff
@@ -44,6 +44,8 @@ export const TxHistory = () => {
       end={{x: isDark ? 0 : 0, y: isDark ? 0.5 : 0}}
       style={styles.root}
     >
+      {Platform.OS === 'android' && <StatusBar translucent backgroundColor="transparent" />}
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/ListTxHistory/useCases/ListTxHistory/TxHistory.tsx]: @banklesss can it be done in the `useStatusBar` ?

Comment (banklesss) [reply apps/wallet-mobile/src/features/ListTxHistory/useCases/ListTxHistory/TxHistory.tsx]: let me check it 👍🏻

Comment (banklesss) [reply apps/wallet-mobile/src/features/ListTxHistory/useCases/ListTxHistory/TxHistory.tsx]: done

**Comment by banklesss** [comment]
converted to draft to make an improvement

**Comment by banklesss** [comment]
done

File: apps/wallet-mobile/src/hooks/useStatusBar.ts [hunk 2]
```diff
@@ -69,8 +77,8 @@ const oldBlueRoutes = ['enable-login-with-os', 'auth-with-os']
  * @warning This function will ignore the alpha channel if it was passed along with the color
  * @warning This function ignore named colors, by ignoring, it means returning the same color passed
  */
-export const simulateOpacity = (color: HexColor): HexColor => {
-  if (!/^#([0-9A-Fa-f]{3}([0-9A-Fa-f]{1})?|[0-9A-Fa-f]{6}([0-9A-Fa-f]{2})?)$/.test(color)) {
+export const simulateOpacity = (color: Color): Color => {
+  if (!isHex(color)) {
```
Comment (stackchain) [inline apps/wallet-mobile/src/hooks/useStatusBar.ts:81]: ty

**Review by stackchain — APPROVED** [review]

**Pull request merged**